### PR TITLE
Add black code formatter

### PR DESCRIPTION
**Pull request by aexvir**
https://github.com/ambv/black/

### CODE REVIEW

File: black/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,13 @@
+FROM python:3.6-alpine3.7
+
+RUN addgroup -S black && adduser -D -S black black && \
+    apk add --no-cache bash && \
```
Comment (chauffer) [inline black/Dockerfile]: Where is this used?

File: black/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+    pip install black==18.4a2 && \
+    mkdir /src
+
+LABEL name=black version=dev \
```
Comment (chauffer) [inline black/Dockerfile]: Version seems to be 18.4a2 here for me, but we nowadays omit both name and version for new images

File: README.md [hunk 3]
```diff
@@ -5,6 +5,13 @@ This is a collection of Dockerfiles maintained by the [Kiwi.com Platform Team](h
 The images are built automatically by Docker Hub for the `:latest` tag,
 with updates of the base image triggering rebuilds.
 
+## kiwicom/black
+
+- Base image: `python:3.6-alpine3.7`
+- Packages: [`black`](https://github.com/ambv/black/)
+
+Image used to format python code and to check if all the files are correctly formatted on CI.
```
Comment (chauffer) [inline README.md]: I would explain that _we_ are using it mostly in CI. However, it's not its only purpose.

I would also add an example usage (`docker run ...`)

File: black/Dockerfile (hunk 1 above)
Comment (aexvir) [reply black/Dockerfile]: What? The user? Is necessary to have a cache directory in it's home. And it's used right [here](https://github.com/kiwicom/dockerfiles/pull/20/files#diff-7c3ef15bbb1c74f6e6c8643822c0a1f5R11)

Comment (chauffer) [reply black/Dockerfile]: bash

Comment (aexvir) [reply black/Dockerfile]: Yep 🤦‍♂️ I was looking at the wrong line. And indeed is not necessary. I don't know why I did that... I tried the image without it and it works fine

**Pull request merged**